### ISSUE TITLE
Admin Profile:  Status message of hours declined not disappearing after refreshing the page. 

### ISSUE THREAD

**Issue by MaxTattenbach** [opencurrents/opencurrents#563]
![image](https://user-images.githubusercontent.com/26234440/31854355-5b5aa6d8-b65d-11e7-8de0-875b40af1459.png)

**Comment by dannypernik** [comment]
@nickolashe I assume the status message is going to stay the same if the URL stays the same? @MaxTattenbach I don't believe there's an easy way to fix this given how we're displaying error messages atm.

**Comment by MaxTattenbach** [comment]
I think we can eliminate that status message. I don't think its that
important. Its pretty evident when hours are declined.

**Comment by dannypernik** [comment]
@MaxTattenbach all status messages will remain when refreshing the page. It's not just an issue with the hours declined message. Also it will say "You have accepted X volunteers and declined Y volunteers" when X and Y are positive. No reason to remove this status message because it persists after page refresh.

**Comment by nickolashe** [comment]
agreed. i don't think that many users will keep refreshing the page. not the most common use case in other words. feel free to close if sounds good

**Comment by MaxTattenbach** [comment]
So does it ever disappear? Or when would it disappear?

Max Tattenbach
Co-creator
www.opencurrents.com

**Comment by dannypernik** [comment]
It would disappear as soon as you click on any link on the page.
